Kernel/Bus/USB/SysFSUSB.cpp

```diff
@@ -162,9 +168,9 @@ UNMAP_AFTER_INIT void SysFSUSBBusDirectory::initialize()
     s_procfs_usb_bus_directory = directory;
 }
 
-NonnullRefPtr<SysFSUSBDeviceInformation> SysFSUSBDeviceInformation::create(USB::Device& device)
+ErrorOr<NonnullRefPtr<SysFSUSBDeviceInformation>> SysFSUSBDeviceInformation::create(USB::Device& device)
 {
-    auto device_name = KString::must_create(String::number(device.address()));
+    auto device_name = TRY(KString::try_create(String::number(device.address())));
     return adopt_ref(*new SysFSUSBDeviceInformation(move(device_name), device));
```

Comment:
you can use `try_make_ref_counted` here and then fail a similar way if we can't allocate `SysFSUSBDeviceInformation` itself

Reply:
Cool, I'll make that change tomorrow :)

Reply:
actually, you can even use `adopt_nonnull_ref_or_enomem` that wraps it into an `ErrorOr` as well (so you can `TRY()` it :)